crates/sui-indexer-alt-schema/migrations/2025-01-16-231500_tx_signatures/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE kv_transactions
+ADD COLUMN IF NOT EXISTS raw_signatures BYTEA NOT NULL;
```

Comment:
raw_user_signatures?

Reply:
(And these are indeed the user signatures -- can change the name).